src/main/kotlin/no/nav/syfo/application/service/VurderingService.kt

```diff
@@ -34,9 +35,10 @@ class VurderingService(
         gjelderFom: LocalDate?,
         svarfrist: LocalDate? = null,
         callId: String,
+        overrideForhandsvarselChecks: Boolean = false,
     ): Vurdering {
         val currentVurdering = getVurderinger(personident).firstOrNull()
-        if (type == VurderingType.FORHANDSVARSEL && currentVurdering is Vurdering.Forhandsvarsel) {
+        if (!overrideForhandsvarselChecks && (type == VurderingType.FORHANDSVARSEL && currentVurdering is Vurdering.Forhandsvarsel)) {
```

Comment:
For å kunne kjøre vanlig prodkode samtidig som vi kan overstyre denne sjekken via cronjobben.